build/NuSpecs/ProjectReunion-Nuget-FrameworkPackage.props

```diff
@@ -0,0 +1,76 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildThisFileDirectory)\MicrosoftProjectReunionVersion.props"/>
+  <Import Project="$(MSBuildThisFileDirectory)\Common.targets"/>
+  <ItemGroup>
+    <AppxPackageRegistration Include="$(MSBuildThisFileDirectory)..\tools\AppX\x86\Release\Microsoft.ProjectReunion.2.5.appx">
+      <Architecture>x86</Architecture>
```

Comment:
This file name needs to match the version. Unfortunately bumping the version today requires updating version.props and this file.

Reply:
so change all the occurrences of Microsoft.ProjectReunion.2.5.appx to M.PR.0.1.appx?

Reply:
Right. I mean ... once you get the FrameworkPackage generation working, you'll see -- the file name it generates is based on the Major.Minor version from version.props. I just never got around to having this particular file be generated by those version numbers, but it should be.

Reply:
thanks, fixed and pushed.